### ISSUE TITLE
Exception on initial install

### ISSUE THREAD

**Issue by bendemboski** [felixrieseberg/ember-electron#326]
`ember install ember-electron` on a new project currently fails with:

```
Project needs forge config
Cannot read property 'replace' of undefined
TypeError: Cannot read property 'replace' of undefined
    at setInitialForgeConfig (/private/tmp/ee-test/node_modules/electron-forge/dist/util/forge-config.js:85:78)
    at /private/tmp/ee-test/node_modules/electron-forge/dist/api/import.js:292:44
    at Generator.next (<anonymous>)
    at Generator.tryCatcher (/private/tmp/ee-test/node_modules/bluebird/js/release/util.js:16:23)
    at PromiseSpawn._promiseFulfilled (/private/tmp/ee-test/node_modules/bluebird/js/release/generators.js:97:49)
    at Promise._settlePromise (/private/tmp/ee-test/node_modules/bluebird/js/release/promise.js:574:26)
    at Promise._settlePromise0 (/private/tmp/ee-test/node_modules/bluebird/js/release/promise.js:614:10)
    at Promise._settlePromises (/private/tmp/ee-test/node_modules/bluebird/js/release/promise.js:693:18)
    at Async._drainQueue (/private/tmp/ee-test/node_modules/bluebird/js/release/async.js:133:16)
    at Async._drainQueues (/private/tmp/ee-test/node_modules/bluebird/js/release/async.js:143:10)
    at Immediate.Async.drainQueues (/private/tmp/ee-test/node_modules/bluebird/js/release/async.js:17:14)
    at runCallback (timers.js:789:20)
    at tryOnImmediate (timers.js:751:5)
    at processImmediate [as _immediateCallback] (timers.js:722:5)
```

This is a bug in `electron-forge` that should be fixed by [this PR](https://github.com/electron-userland/electron-forge/pull/372). Filing this issue for tracking purposes, and in case anybody runs into it. The workaround is to add  `productName` to your `package.json` (even if it's empty):

```
"productName": ""
```

**Comment by anulman** [comment]
@bendemboski @malept See this got merged in the last couple of days. Any ETA re release?

**Comment by malept** [comment]
@anulman thanks for reminding me, I just released Electron Forge 4.1.6.

**Comment by anulman** [comment]
Since package.json defines `"electron-forge": "~4.1.1"`, I think we're good to go 👍

**Comment by ksolo** [comment]
@bendemboski 
I'm getting a similar error on install:

```bash
Project needs forge config
Failed to install modules: ["babel-preset-env","babel-preset-react","babel-plugin-transform-async-to-generator","electron-forge"]

With output: Exited with status 1
```

Is this related the this issue or should I open a new one? I did add a `productName` key to the `package.json` as mentioned and it is still failing.

Thanks for taking a look.

**Comment by malept** [comment]
@ksolo your error is unrelated.

**Comment by malept** [comment]
@bendemboski I'd reword the summary similar to the submitted Forge PR title to avoid future confusion.

**Comment by ksolo** [comment]
@malept I'll open a new issue.
